backend/lib/spree/backend_configuration.rb

```diff
@@ -21,6 +21,10 @@ def theme_path(user_theme = nil)
       user_theme ? themes.fetch(user_theme.to_sym) : themes.fetch(theme.to_sym)
     end
 
+    # @!attribute [rw] admin_updated_navbar
+    #   @return [Boolean] Should the updated navbar be used in admin (default: +false+)
+    versioned_preference :admin_updated_navbar, :boolean, initial_value: false, boundaries: { "4.1.0.a" => true }
```

Comment:
I'm a bit concerned that we will forget to update this. Any way to force us to remember this before the `admin` branch will be merged in `main`? Maybe we can remove the boundaries?

Reply:
@kennyadsl you mean that we'll need to move the boundary to 4.2 or beyond or remove them completely? 
Anyway I will push for a final review of all the code in the admin dev branch before it's merged to main, especially the code outside of `admin/`

Reply:
> you mean that we'll need to move the boundary to 4.2 or beyond or remove them completely?

We will need to specify the exact version from which we want that preference to be suggested during the upgrade (via the `new_solidus_defaults.rb` initializer), and will be the default on new installations. 

If we plan to re-review everything, I think it's ok, my point was related to if we have ways to be 99% sure we catch it during that review. Maybe a TODO comment? BTW, if you feel confident that it's not an issue, we can merge it as is.

Reply:
I'll add a TODO comment, seems good enough as a reminder 👍